pkg/local_object_storage/blobstor/fstree/fstree.go

```diff
@@ -477,3 +477,27 @@ func (t *FSTree) SetCompressor(cc *compression.Config) {
 func (t *FSTree) SetReportErrorFunc(_ func(string, error)) {
 	// Do nothing, FSTree can encounter only one error which is returned.
 }
+
+// CleanUpTmp removes all temporary files garbage.
+func (t *FSTree) CleanUpTmp() error {
+	err := filepath.WalkDir(t.RootPath,
+		func(path string, d fs.DirEntry, err error) error {
+			if err != nil {
+				return err
+			}
+			if !d.IsDir() && strings.Contains(d.Name(), "#") {
```

Comment:
```suggestion
			if !d.IsDir() && strings.HasPrefix(d.Name(), "#") {
```
?

Reply:
According to this [tmp path](https://github.com/nspcc-dev/neofs-node/blob/b5a9d38c08032da4f1bbe1edf3a2889f9d6b3847/pkg/local_object_storage/blobstor/fstree/fstree_write_generic.go#L61), I think that the path would be like `/root/path/cid.oid#retryCount`, so `#` in the middle of file name, Am I not right?

Reply:
i think `Name()` should not return the whole path:
```
	// Name returns the name of the file (or subdirectory) described by the entry.
	// This name is only the final element of the path (the base name), not the entire path.
	// For example, Name would return "hello.go" not "home/gopher/hello.go".
```

Reply:
oh, you mean the name itself is also does not start with `#`, i see